predictive.py

```diff
@@ -17,61 +17,133 @@
 
 # Predictive Transformer
 class PredictiveTransform:
+    """
+    This class encodes or decodes a 2D array using a predictive transformation. The main objective of this algorithm is to reduce entropy of the
+    given information.
 
+    Use only encode() or decode()
+    """
     def __init__(self):
+        """
+        Constructor. It is empty
```

Comment:
This is a weak description! Correct it please

Reply:
No u